### PR TITLE
Cgi/transcoding/ffmpeg tools

### PR DESCRIPTION
**Pull request by nieznanysprawiciel**

### CODE REVIEW

**Review by bidzyyys — APPROVED** [review]
:+1:

File: apps/transcoding/ffmpeg/resources/ffmpeg-scripts/ffmpeg_task.py [hunk 1]
```diff
@@ -111,7 +113,11 @@ def run():
     with open(PARAMS_FILE, 'r') as f:
         params = json.load(f)
 
-    run_ffmpeg(params)
+    try:
+        run_ffmpeg(params)
+    except commands.CommandFailed as e:
+        print(e.command)
```
Comment (weaselix) [inline apps/transcoding/ffmpeg/resources/ffmpeg-scripts/ffmpeg_task.py]: print it to `stderr`, please

File: apps/transcoding/task.py [hunk 2]
```diff
@@ -74,30 +78,54 @@ def __setstate__(self, state):
 
     def initialize(self, dir_manager: DirManager):
         super(TranscodingTask, self).initialize(dir_manager)
-        logger.debug('Initialization of ffmpegTask')
+
+        logger.debug('Initialization of FFmpegTask')
+
         task_id = self.task_definition.task_id
         task_output_dir = dir_manager.get_task_output_dir(task_id)
+        input_file = self.task_resources[0]
```
Comment (weaselix) [inline apps/transcoding/task.py]: You did it **before** actual checking of content. Please look at `90` line.

File: apps/transcoding/ffmpeg/resources/ffmpeg-scripts/ffmpeg_task.py (hunk 1 above)
Comment (nieznanysprawiciel) [reply apps/transcoding/ffmpeg/resources/ffmpeg-scripts/ffmpeg_task.py]: Changed.
But I don't update ffmpeg image on dockerhub.
This change will be introduced next time when other big changes will be made to this image.

Comment (weaselix) [reply apps/transcoding/ffmpeg/resources/ffmpeg-scripts/ffmpeg_task.py]: Yes, you right.

File: apps/transcoding/task.py (hunk 2 above)
Comment (nieznanysprawiciel) [reply apps/transcoding/task.py]: Done

**Pull request merged**